workers/worker_base.py

```diff
@@ -454,43 +466,75 @@ def find_id_from_login(self, login):
+                "gh_node_id": contributor['node_id'],
+                "gh_avatar_url": contributor['avatar_url'],
+                "gh_gravatar_id": contributor['gravatar_id'],
+                "gh_followers_url": contributor['followers_url'],
+                "gh_following_url": contributor['following_url'],
+                "gh_gists_url": contributor['gists_url'],
+                "gh_starred_url": contributor['starred_url'],
+                "gh_subscriptions_url": contributor['subscriptions_url'],
+                "gh_organizations_url": contributor['organizations_url'],
+                "gh_repos_url": contributor['repos_url'],
+                "gh_events_url": contributor['events_url'],
+                "gh_received_events_url": contributor['received_events_url'],
+                "gh_type": contributor['type'],
+                "gh_site_admin": contributor['site_admin'],
+                "tool_source": self.tool_source,
+                "tool_version": self.tool_version,
+                "data_source": self.data_source
+            }
+        
+        elif platform == 'gitlab':
+            cntrb =  {
+                "cntrb_login": contributor[0]['username'] if 'username' in contributor[0] else None,
+                "cntrb_email": email,
+                "cntrb_company": company,
+                "cntrb_location": location,
+                "cntrb_created_at": contributor[0]['created_at'] if 'created_at' in contributor[0] else None,                
+                "cntrb_canonical": None,
+                "gh_user_id": contributor[0]['id'],
+                "gh_login": contributor[0]['username'],
+                "gh_url": contributor[0]['web_url'],
+                "gh_html_url": None,
+                "gh_node_id": None,
+                "gh_avatar_url": contributor[0]['avatar_url'],
+                "gh_gravatar_id": None,
+                "gh_followers_url": None,
+                "gh_following_url": None,
+                "gh_gists_url": None,
+                "gh_starred_url": None,
+                "gh_subscriptions_url": None,
+                "gh_organizations_url": None,
+                "gh_repos_url": None,
+                "gh_events_url": None,
+                "gh_received_events_url": None,
+                "gh_type": None,
+                "gh_site_admin": None,
+                "tool_source": self.tool_source,
+                "tool_version": self.tool_version,
+                "data_source": self.data_source
+            }
         result = self.db.execute(self.contributors_table.insert().values(cntrb))
         self.logger.info("Primary key inserted into the contributors table: " + str(result.inserted_primary_key))
         self.results_counter += 1
         self.cntrb_id_inc = int(result.inserted_primary_key[0])
 
-        self.logger.info("Inserted contributor: " + contributor['login'] + "\n")
+        logging.info("Inserted contributor: " + contributor[0]['username'] + "\n")
```

Comment:
Please make sure you use self.logger.info instead of logging.info, sorry I know this is a recent change.